qubesbuilder/plugins/installer/Makefile

```diff
@@ -26,7 +26,9 @@ INSTALLER_DIR ?= $(PLUGINS_DIR)/installer
 SOURCES_DIR ?= $(BUILDER_DIR)/sources
 INSTALLER_KICKSTART ?= $(SOURCES_DIR)/qubes-release/conf/qubes-kickstart.cfg
 CREATEREPO := $(shell which createrepo_c createrepo 2>/dev/null |head -1)
+
 ISO_VERSION ?= $(shell date +%Y%m%d)
+ISO_VERSION ?= $(shell date -d @"$(SOURCE_DATE_EPOCH)" +%Y%m%d 2>/dev/null || date +%Y%m%d)
```

Comment:
`ISO_VERSION` is set already on the line just above. I think you wanted to replace that one, not add another one

Reply:
Same comment than template applies here too. We cannot reproduce a given ISO or template based on timestamp. We don't hold any snapshot at a given point of time.